AdobeDocs/commerce-pwa-studio
# Complete migration of tutorials topics

## Description

This PR contains the following changes:

- Adds a linter ([remark-lint][]) to enforce some basic standards
- Adds prettier to fix formatting
- Fixes the links in the previously migrated Tutorial topics
- Fixes linting and formatting issues using the new linter and prettier scripts

[remark-lint]: https://github.com/remarkjs/remark-lint

## Related Issue

Closes PWA-1621

## Motivation and Context

Linter and prettier added to ensure quality and format standardization.
Content updated to fix linking and theme compatibility.

## How Has This Been Tested?

1. Test formatting with prettier: `npm run prettier:test`
2. Verify no prettier errors
3. Test markdown standards with remark-lint: `npm run lint`
4. Verify only one warning (fixing this requires out-of-scope changes to the linter)
5. Run the dev server: `npm run dev`
6. Verify server builds and runs site
7. Verify tutorial pages exists and looks correct
8. Run a build: `npm run build`
9. Verify build succeeds
10. Run an HTML server on the static assets: `npm run serve`
11. Verify site works

## Screenshots (if appropriate):

## Types of changes




## Checklist:
